tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -47,9 +51,9 @@ class TransactionsSpecTest extends FunctionalTestCase
         'transaction-options: readConcern snapshot in startTransaction options' => 'PHPLIB does not properly inherit readConcern for transactions',
     ];
 
-    private static function doSetUpBeforeClass()
+    private function doSetUp()
     {
-        parent::setUpBeforeClass();
+        parent::setUp();
```

Comment:
This change makes the test more compliant with the spec, which requires that all sessions be killed before running a test. We still kill all sessions after a failing test.